source/interprocedural_analyses/taint/taintConfiguration.ml

```diff
@@ -412,7 +453,15 @@ let parse source_jsons =
         in
         { literal_strings }
   in
-
+  let check_json json =
+    check_keys
+      ~required_keys:["sources"; "sinks"; "features"; "rules"]
```

Comment:
I don't think the root taint.config has any required keys, actually. Features definitely aren't required, and we have some hardcoded sources/sinks/rules for testing so this shouldn't be necessary either.

Reply:
Could we add this back but without the required keys? It's still useful to know if we have unknown keys at the root.